.github/workflows/ci.yml

```diff
@@ -37,14 +37,14 @@ jobs:
     steps:
       - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # tag=v3.1.0
 
-      - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # tag=v1.0.7
+      - uses: actions-rs/toolchain@88dc2356392166efad76775c878094f4e83ff746
         with:
           profile: minimal
-          toolchain: nightly
+          toolchain: nightly-2022-09-22
```

Comment:
Oh, and also, why pinning to `nightly-2022-09-22`?

Reply:
yea.. to prevent "not supported" in latest cargo
`2022-09-22` is safe
![image](https://user-images.githubusercontent.com/8097901/203458959-8a8b0150-4359-4817-8cba-0cadee93e6fa.png)